apps/platform-app/src/app/[locale]/page.tsx

```diff
@@ -17,16 +17,18 @@ export default async function HunterLandingPage() {
   return (
     <main className={styles.hunter_landing}>
       <Container>
-        <HunterHeader />
-        <HunterHero />
-        <HunterRiskReducers />
-        <HunterHowItWorks />
-        <HunterWhyWorth />
-        <HunterReviews />
-        <HunterTeam />
-        <HunterCTA />
-        <HunterFAQ />
-        <HunterFooter />
+        <div style={{ display: 'flex', flexDirection: 'column', gap: '180px' }}>
+          <HunterHeader />
```

Comment:
Move <hunterHeader/> and <hunterFooter> outside the <main/> tag to improve semantics